packages/cms/src/index.js

```diff
@@ -1,22 +1,36 @@
-export {default as Builder} from "./Builder.jsx";
+import {chunkify} from "@datawheel/canon-core";
 
 // Components
-export {default as Profile} from "./components/Profile.jsx";
-export {default as Mirror} from "./components/Viz/Mirror.jsx";
-export {default as Story} from "./components/Story.jsx";
-export {default as StoryLanding} from "./components/StoryLanding.jsx";
-export {default as Section} from "./components/sections/Section.jsx";
+const Builder = chunkify(import(/* webpackChunkName: "cms-builder" */ "./Builder.jsx"));
+export {Builder};
+
+const Profile = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/Profile.jsx"));
+const Mirror = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/Viz/Mirror.jsx"));
+const Section = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/sections/Section.jsx"));
+const PDFButton = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/sections/components/PDFButton.jsx"));
+const Selector = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/sections/components/Selector.jsx"));
+const Stat = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/sections/components/Stat.jsx"));
+const Viz = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/Viz/Viz.jsx"));
+const StatGroup = chunkify(import(/* webpackChunkName: "cms-profile" */ "./components/Viz/StatGroup.jsx"));
+export {
+  Profile,
+  Mirror,
+  Section,
+  PDFButton,
+  Selector,
+  Stat,
+  Viz,
+  StatGroup
+};
+
+const Story = chunkify(import(/* webpackChunkName: "cms-story" */ "./components/Story.jsx"));
+const StoryLanding = chunkify(import(/* webpackChunkName: "cms-story" */ "./components/StoryLanding.jsx"));
```

Comment:
For the reasons I explained in the chunkify function, these `import()` calls load all the components on page load:
![image](https://user-images.githubusercontent.com/11713191/97230949-8a343d00-17b9-11eb-83e5-eeb6cd438f4f.png)
This is the index page for the example app, which doesn't import any of these, but loads them by importing anything from `@datawheel/canon-cms`.

Reply:
Fixed to use the correct `() => import()` argument syntax.